cfgov/templates/wagtailadmin/login.html

```diff
@@ -64,10 +61,14 @@ <h1>Log in to consumerfinance.gov</h1>
                         </div>
                     </li>
                 {% endcomment %}
+                {% endblock %}
                 <li class="submit">
+                    {% block submit_buttons %}
                     <button type="submit" class="button button-longrunning" tabindex="3" data-clicked-text="{% trans 'Signing in...' %}"><span class="icon icon-spinner"></span><em>{% trans 'Sign in' %}</em></button>
+                    {% endblock %}
                 </li>
             </ul>
+        {% endblock %}
```

Comment:
Can we bring in [this logic](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailadmin/templates/wagtailadmin/login.html#L82) to add the username focus?

Reply:
I *think* that gets brought over because the template [`overextends`](https://github.com/cfpb/cfgov-refresh/pull/4084/files/b99dd9c4ca1c8b33f62326f9e5d5717453783fd2#diff-c0e993a61a4307b99a36af5801d4e183L1) wagtailadmin/login.html? I will confirm.

Reply:
@willbarton I can confirm that this does get brought in.